bip-0342.mediawiki

```diff
@@ -100,7 +100,7 @@ The following rules apply to <code>OP_CHECKSIG</code>, <code>OP_CHECKSIGVERIFY</
 *** For <code>OP_CHECKSIG</code>, an empty vector is pushed onto the stack, and execution continues with the next opcode.
 *** For <code>OP_CHECKSIGADD</code>, a <code>CScriptNum</code> with value <code>n</code> is pushed onto the stack, and execution continues with the next opcode.
 ** If the signature is not the empty vector, the opcode is counted towards the sigops budget (see further).
-*** For <code>OP_CHECKSIGVERIFY</code>, execution continues without any further changes to the stack.
+*** For <code>OP_CHECKSIGVERIFY</code>, a an element is popped from the stack. 
```

Comment:
"a an element" and trailing whitespace.

Note that the first bullet point here already says "For <code>OP_CHECKSIGVERIFY</code> and <code>OP_CHECKSIG</code>, the public key (top element) and a signature (second to top element) are popped from the stack."

Maybe it would be better to describe the difference between "OP_CHECKSIG OP_VERIFY" and "OP_CHECKSIGVERIFY" -- in particular, the only case where they differ is if `stack.size() + altstack.size() = MAX + 1` after `OP_CHECKSIG` completes, in which case the former will always fail, while the latter may succeed.

Reply:
> Note that the first bullet point here already says

Ah, you're right I'd missed that! This section decribes what happens after the initial pops and verification. So in this case `OP_CHECKSIGVERIFY` does just indeed "continue as normal".

Reply:
dana.</ref>.